src/protocol/java/src/main/java/org/apache/jmeter/protocol/java/sampler/JavaSampler.java

```diff
@@ -121,9 +121,9 @@ private void initClass() {
             javaClass = Class.forName(name, false, Thread.currentThread().getContextClassLoader());
             Method method = javaClass.getMethod("teardownTest", JavaSamplerContext.class);
             isToBeRegistered = !method.getDeclaringClass().equals(AbstractJavaSamplerClient.class);
-            log.info("Created class: {}. Uses tearDownTest: ", name, isToBeRegistered);
+            log.info("Created class: {}. Uses tearDownTest: {}", name, isToBeRegistered);
         } catch (Exception e) {
-            log.error("{}\tException initialising: ", whoAmI(), name, e);
+            log.error("{}\tException initialising: {} {}", whoAmI(), name, e);
```

Comment:
```suggestion
            log.error("{}\tException initialising: {}", whoAmI(), name, e);
```

Reply:
I don't quite understand the suggested change. Why are you removing the third placeholder? The signature of the called method is 
public void error(String format, Object... arguments);
You will lose data about exception

Reply:
The exception should not be among the placeholders.
If you add a placeholder for the exception, then then the logger prints only the exception class.
If you omit the placeholder for the exception, then the logger prints the full stacktrace

Reply:
Here's the documentation: https://www.slf4j.org/faq.html#paramException